docker/build/build-images.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# build base image
+docker inspect --type=image nautilus-cloud:base-latest &>/dev/null || ./docker/base/build-base-image.sh
+# 2-stage build in docker 
+docker build -t nautilus-cloud -f ./docker/build/dockerfile-build .
```

Comment:
is the path to the `dockerfile-build` correct, relative to where this script will be expected to run from?

Reply:
yup. as per instructions you should be in the root directory of the repo.